edkahara/iReporter
# Ft create v2 delete report endpoint #162423328

__What does this PR do__
Create the endpoint that allows a user to delete a report.

__Description of tasks to be completed.__
Create an api endpoints that facilitates the deleting of a report. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-delete-report-endpoint-#162423328 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set up environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetests.

__Relevant pivotal tracker stories.__
#162423328
